src/scss/components/contact.scss

```diff
@@ -0,0 +1,38 @@
+.contact {
+  @extend .container;
+  padding-left: 19px;
+  padding-right: 19px;
+
+  &__title {
+    font-family: $hkgrotesk-bold;
+    font-size: $mobile-title;
+    margin-bottom: 20px;
+    text-align: center;
+  }
+
+  &__paragraph {
+    color: $gray;
+    font-family: $hkgrotesk-regular;
+    font-size: $mobile-paragraph;
+    margin-bottom: 40px;
+    text-align: center;
+  }
+
+  &__label {
+    font-family: $hkgrotesk-bold;
+    font-size: $desktop-card-titles;
+    margin-bottom: 10px;
+  }
+
+  &__text {
+    color: $gray;
+    font-family: $hkgrotesk-regular;
+    font-size: $mobile-paragraph;
+    padding-bottom: 50px;
+    //width: 162px;
```

Comment:
If you don't need this anymore, please remove it. Do not leave the commented code.